database/Cargo.toml

```diff
@@ -25,3 +25,7 @@ actix-diesel = { git = "https://github.com/frol/actix-diesel", rev = "3a001986c8
 near-indexer-primitives = "0.14"
 near-primitives = "0.14"
 near-crypto = "0.14"
+
+[features]
+default = ["load_fungible_token_events"]
```

Comment:
It's switched on by default, right?
Let's make the opposite so that the users need to add the flag for collecting this data

Reply:
Sure, done